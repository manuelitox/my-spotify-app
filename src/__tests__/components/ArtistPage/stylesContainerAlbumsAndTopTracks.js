import React from 'react'
import { shallow } from 'enzyme'

import StyledContainerAlbumsAndTopTracks from 'components/ArtistPage/stylesContainerAlbumsAndTopTracks'

describe('Component | StyledContainerAlbumsAndTopTracks', () => {
  let component 
  beforeEach( () => {
    component = shallow(
      <StyledContainerAlbumsAndTopTracks />
    )
  })
  it('should render a styled component', () => {
    expect(component).toMatchSnapshot()
  })
})