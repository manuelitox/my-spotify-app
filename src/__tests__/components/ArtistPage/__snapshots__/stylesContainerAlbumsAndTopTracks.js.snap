// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component | StyledContainerAlbumsAndTopTracks should render a styled component 1`] = `
<styled.div>
  <styled.div />
</styled.div>
`;
